sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## 7.0.0b3 (Unreleased)
 
+**New Features**
+
+* Added methods `get_queue_deadletter_receiver` and `get_subscription_deadletter_receiver` in `ServiceBusClient` to get a `ServiceBusReceiver` for the sub dead-letter-queue of the target entity.
+
+**BugFixes**
+
+* Fixed bug where reason and description not being set when dead-lettering messages.
```

Comment:
```suggestion
* Fixed bug where reason and description were not being set when dead-lettering messages.
```

Really bikesheddily-small nit.  I'll use this space to ponder if we wanted to note the UAMQP bump; it's somewhat moot since we're still in preview mode, but to carry over whatever consistency we're trying to hold up in the VOLd where we noted the version change.

Reply:
updated